examples/metrics/Makefile

```diff
@@ -2,7 +2,7 @@ CONTAINER_RUNTIME := docker
 
 .PHONY: binary-container
 binary-container:
-	${CONTAINER_RUNTIME} build -f build/Dockerfile -t zot-build:latest ../../.
+	${CONTAINER_RUNTIME} build -f Dockerfile -t zot-build:latest ../../.
```

Comment:
Our Dockerfiles are located under build/ right?

Reply:
Not anymore. Not sure who/why removed them from there (can be investigated)

Reply:
Are we discussing https://github.com/project-zot/zot/blob/main/build/Dockerfile? Or others?

Reply:
https://github.com/project-zot/zot/tree/main/build
^ i still see them?